lib/utilities/detect-deprecated-config.js

```diff
@@ -0,0 +1,8 @@
+var path = require('path');
+var glob = require('glob');
+
+module.exports = function() {
+  var deprecatedConfigPath = path.join(__dirname, '../../../..', 'config', 'deploy', '*.js');
```

Comment:
I'm fairly certain that `path.join` will normalize the '/' for you, so there's no need to artificially break the segments up as separate arguments. And indeed, I confirmed this is so (at least on Win11/Node 0.10 and 0.12)

![screen shot 2015-03-06 at 9 31 24 am](https://cloud.githubusercontent.com/assets/119972/6526888/54e60f2c-c3e4-11e4-9dcf-71414373d59b.png)